pkg/controller/dataimportcron-controller.go

```diff
@@ -771,14 +772,27 @@ func (r *DataImportCronReconciler) newCronJob(cron *cdiv1.DataImportCron) (*batc
 		TerminationMessagePolicy: corev1.TerminationMessageReadFile,
 	}
 
+	volumes := []corev1.Volume{}
 	hasCertConfigMap := regSource.CertConfigMap != nil && *regSource.CertConfigMap != ""
 	if hasCertConfigMap {
 		vm := corev1.VolumeMount{
 			Name:      CertVolName,
 			MountPath: common.ImporterCertDir,
 		}
-		container.VolumeMounts = []corev1.VolumeMount{vm}
+		container.VolumeMounts = append(container.VolumeMounts, vm)
 		container.Command = append(container.Command, "-certdir", common.ImporterCertDir)
+		volumes = append(volumes, createConfigMapVolume(CertVolName, *regSource.CertConfigMap))
```

Comment:
So prior to this PR we did not support custom CA for polling the dataimportcron registry?
(This is unrelated to the proxy stuff, right?)
I just want to make sure I get this right

Reply:
We used certs but not proxy certs. See it in the refactored code here.

Reply:
I mean that before, we passed the `regSource.CertConfigMap`, but we did not capitalize on it in the poller
(just polled regularly)

Reply:
We already used the regular certs in the poller before (see GetImageDigest implementation):
https://github.com/kubevirt/containerized-data-importer/blob/main/tools/cdi-source-update-poller/main.go#L51

Reply:
So I am missing basically the equivalent of https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/controller/import-controller.go#L601-L617 in the job configuration. The environment variables is what enables the proxy to work.

Reply:
sure, re-adding, and adding test coverage as well.